application/src/test/java/org/opentripplanner/street/model/edge/StreetEdgeGeofencingTest.java

```diff
@@ -257,9 +257,9 @@ public void pickupFloatingVehiclesWhenStartedInNoDropOffZone() {
 
       var states = edge.traverse(haveRentedState);
 
-      // we return 4 states: one for continuing walking, one for the speculative renting of
+      // we return 3 states: one for continuing walking, one for the speculative renting of
```

Comment:
In the meeting today we said that we would like separate test cases for the scenarios.